support/COMMUNITY_CONTACTS.md

```diff
@@ -0,0 +1,73 @@
+# Duty
+
+Every week we dedicate one individual (the community contact) to watch for user
+issues and either answer them or redirect the questions to other who can. The
+community contact's duty (subject to change) is as followed:
+
+## Daily check list
+
+* Check the knative-users@ mailing list for new unanswered questions.
+* Check [unassigned issues / untriaged
+  issues](https://github.com/knative/serving/issues?q=is%3Aopen+is%3Aissue+no%3Aassignee+no%3Amilestone)
+  for unanswered questions.
+* Join conversations that @serving-help is tagged.
```

Comment:
How do you do that?

Reply:
CF community has a slack bot users tag ie. `@interrupt` that bot creates a subthread and @ the specific user who's on call. The lookup is done in a separate system.
 
https://github.com/cloudfoundry/slack-interrupt-bot

We could make a similar bot and do schedule lookup based on what's on github.

Reply:
@dprotaso I think that would be nice, but we could start by manually adding people in the mean time to kick start the rotation. We can follow up with more automations.

Reply:
@vagababov Added a link.